src/components/Card/CardList.test.tsx

```diff
@@ -48,6 +48,10 @@ describe("CardList component", () => {
     );
   });
 
+  it("adds a card link to localstorage when card item is swiped", () => {
```

Comment:
ChatGPT will help write this for you 😅

Reply:
The tests ChatGPT offered were not quite right. I think I need to sit and properly go through a React testing tutorial. I will try to make time for that next week.

Reply:
! I'm surprised. It gave me helpful structures and methods that I wouldn't have known about, and it was just a matter of integrating it with what I kinda knew already. I haven't been writing many React tests though so would love to see how you approach this.